js/main.js

```diff
@@ -23,6 +23,7 @@ var TIME_NOW = [now.getHours(), now.getMinutes()];
 var DAY_INDEX = (now.getDay() + 6) % 7;  // In our data, first day is Monday
 var DAY_NAMES = ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'];
 var DEFAULT_MARKET_TITLE = 'Markt';
+var firstTimeLoad = true; // indicates if the map was loaded for the first time
```

Comment:
@dirkschumacher How about `initialPageLoad` or `initialMapLoad` or `initialDomLoad` and omitting the comment? Please also rebase your branch onto the latest commit on `master`.